app/src/main/java/org/wikipedia/history/HistoryEntry.kt

```diff
@@ -81,6 +81,7 @@ class HistoryEntry(
         const val SOURCE_APP_SHORTCUT_RANDOM = 20
         const val SOURCE_APP_SHORTCUT_CONTINUE_READING = 21
         const val SOURCE_FEED_MOST_READ_ACTIVITY = 22
+        const val SOURCE_FEED_PLACES = 42
```

Comment:
Can we move this to the bottom of the list, so that they are in numerical order?

Reply:
Yes.